makefiles/simulators/Makefile.modelsim

```diff
@@ -43,7 +43,7 @@ VSIM_ARGS += -onfinish exit
 endif
 
 ifeq ($(GPI_IMPL),vhpi)
-VSIM_ARGS += -foreign \"cocotb_init libfli.so\" -trace_foreign 3
+VSIM_ARGS += -foreign \"cocotb_init libfli.$(LIB_EXT)\" -trace_foreign 3
```

Comment:
Not a comment on this commit, but this has reminded me that we left debug on for FLI!